pkg/controller/operconfig/operconfig_controller.go

```diff
@@ -301,6 +302,16 @@ func (r *ReconcileOperConfig) Reconcile(ctx context.Context, request reconcile.R
 			err = errors.Wrapf(err, "could not apply (%s) %s/%s", obj.GroupVersionKind(), obj.GetNamespace(), obj.GetName())
 			log.Println(err)
 
+			// If error comes from nonexistent namespace print out a help message.
+			if obj.GroupVersionKind().Kind == "NetworkAttachmentDefinition" && strings.Contains(err.Error(), "namespaces") {
+				log.Println(" ")
```

Comment:
Hey Nikhil, this error message probably looks really nice in a terminal, and I'm all about that. However, I think by convention, the go error

Thanks regardless for making this actionable output, I really like that it gives the users steps to take.

But I think we want to condense this into a single line, even if it's a big long. Also, maybe prefix it with some more information about the context of the error, for example:

```
namespace error for networkattachment definition, consider these possible solutions: ...etc
```